corehq/apps/domain/forms.py

```diff
@@ -402,6 +403,12 @@ def clean_sub_area(self):
 
 class DomainGlobalSettingsForm(forms.Form):
     hr_name = forms.CharField(label=_("Project Name"))
+
+    alias = forms.CharField(
+        required=False,
+        label=_("Project Alias"),
```

Comment:
Updating. Should all of the strings in DomainGlobalSettingsForm be _lazy/_noop? (I read ReadTheDocs' bit on translation, but I don't understand why this class uses _noop for the time zone and _ for all other labels and help...aren't all strings here outside of user/session scope?)